docs/flow/modin/core/execution/dispatching.rst

```diff
@@ -52,3 +52,8 @@ which in turn forwards this call to the actual implementation of ``read_csv`` 
 ``PandasOnRayIO.read_csv``. The result of ``modin.pandas.read_csv`` will return a high-level Modin
 DataFrame with the appropriate `QueryCompiler` bound to it, which is responsible for
 dispatching all of the further function calls.
+
+Factory examples
+================
+- .. autoclass:: modin.core.execution.dispatching.factories.factories.PandasOnRayFactory
+- .. autoclass:: modin.core.execution.dispatching.factories.factories.ExperimentalPandasOnRayFactory
```

Comment:
These `autoclass` directives have been added here and elsewhere to make `:py:class:` directives work (to make the links clickable).

Reply:
It's strange that only these classes are presented. I would prefer to see this section called something like "Supported factories" storing `autoclass`es for every factory presented in Modin. This change seems unrelated to the PR though. I propose we add this section separately.

For now, we can either leave things as it is - unclickable factory names, that will automatically become links when we add the described section or just make the factory's hyperlink pointing to the `dispatching.rst` file.

Reply:
That's why I called the section - "examples" :)
Why not leave it as it is, so that the links work? It will be possible to supplement it later.

Reply:
The first suggestion looks good to me.

Reply:
I agree with @dchigarev. All factories should be presented, but this is a part of separate PR.

Reply:
@dchigarev @prutskov @YarShev such a decision will worsen one thing, while not at all improving another. I don’t understand why it’s better to do this.

Reply:
That looks as if we have only these factories/IO classes/parsers. I suggest adding those as part of a separate issue. Create an issue for this please.

Reply:
What about this?

Reply:
If you *definitely* don't like having only a subset of these classes, then let me add them here. This is a fairly simple change.

Reply:
That change doesn't fully relate to the issue to be resolved in this PR and should be done in a separate PR because we have factories not only for ExperimentalPandasOnRay. We should keep commit history clean.

Reply:
Indeed, but there is simply a difference in our views. It's okay for me when simple changes are made in the same PR if there is even an indirect relationship.

Reply:
We should be strict as much as possible with respect to any change.

Reply:
ok